apps/coordinator/package.json

```diff
@@ -61,6 +61,7 @@
     "jest-junit": "^11.0.1",
     "prettier": "^3.2.5",
     "standard-version": "^9.0.0",
+    "ts-jest": "^29.3.2",
```

Comment:
wonder if we could keep the version teh same as the removed one so that the package-lock change is not as different.